.github/workflows/node.yml

```
@@ -269,3 +271,45 @@ jobs:
                       node/test-report*.html
                       utils/clusters/**
                       benchmarks/results/**
+
+    test-modules:
+        if: github.event.pull_request.head.repo.owner.login == 'valkey-io'
+        # environment: AWS_ACTIONS
+        name: Running Module Tests
+        runs-on: [self-hosted, linux, ARM64]
+        timeout-minutes: 15
+
+        steps:
+            - name: Setup self-hosted runner access
+              run: sudo chown -R $USER:$USER /home/ubuntu/actions-runner/_work/valkey-glide
+
+            - uses: actions/checkout@v4
+              with:
+                  submodules: recursive
+
+            - name: Use Node.js 16.x
+              uses: actions/setup-node@v3
```

Update to most recent version (is it v4 or v5?) to avoid warning.
```suggestion
              uses: actions/setup-node@v4
```